src/yaml/fullLayerMetadata.yaml

```diff
@@ -32,3 +32,22 @@ components:
               - image/png
               - image/jpeg
               description: Image format associated with content type (mime)
+            minResolutionDeg:
+              type: number
+              minimum: 1.67638e-7
```

Comment:
R U sure about this number?

Reply:
yes, i checked but updated to the exact value from mc-utils